app/src/main/java/app/tivi/showdetails/DetailsViewModelModule.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.showdetails
+
+import com.squareup.inject.assisted.dagger2.AssistedModule
+import dagger.Module
+
+@AssistedModule
+@Module(includes = [AssistedInject_DetailsViewModelModule::class])
```

Comment:
This works? I've always gotten "cannot find symbol" errors when trying to reference generated code in an annotation o_O

Reply:
@hzsweers I see the same error with Dagger 2.17, if you drop back down to 2.16 it goes away. Not sure why yet. Tivi doesn't build if you upgrade to 2.17.